### PR TITLE
Draft: Add XR_FB_Passthrough support

### PR DESCRIPTION
**Pull request by pierre-l**
Still a lot of rough edges:

- needs more doc,
- depends on https://github.com/Ralith/openxrs/pull/112
- needs to be optional (hide it behind a feature?)
- there's an ugly `transmute` that I really need to get rid of

### CODE REVIEW

File: examples/passthrough/scripts/run_on_device.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+adb shell am force-stop rust.passthroug_example
+
+Set-Location $PSScriptRoot\..
+cargo apk run --release
+
+if ($?) {
+    Start-Sleep -Seconds 2
+    $processIdStr = (adb shell pidof rust.passthroug_example) | Out-String
```
Comment (pierre-l) [inline examples/passthrough/scripts/run_on_device.ps1:8]: Typo

Comment (pierre-l) [reply examples/passthrough/scripts/run_on_device.ps1:8]: And same at line 1: passthroug+h

**Comment by kanerogers** [comment]
Hey @pierre-l, sorry I haven't had a chance to look at this yet!

Now that @Ralith has merged in https://github.com/Ralith/openxrs/pull/112, I'll take a look at this next week. 😄

**Comment by kanerogers** [comment]
@pierre-l Looks like clippy is a little unhappy about this one. Any chance you could take a look at the requested change? :)

**Comment by kanerogers** [comment]
> @pierre-l Looks like clippy is a little unhappy about this one. Any chance you could take a look at the requested change? :)

Ignore: this has now been updated, update the branch when ready!

**Comment by pierre-l** [comment]
My experiments yielded unsatisfying results, leaving me without incentive to complete this PR.

**Pull request closed without merging**